main/tests/server/src/com/google/refine/commands/column/AddColumnCommandTests.java

```diff
@@ -0,0 +1,56 @@
+
+package com.google.refine.commands.column;
+
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
+
+import java.util.Collections;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.google.refine.browsing.Engine;
+import com.google.refine.browsing.EngineConfig;
+import com.google.refine.model.AbstractOperation;
+import com.google.refine.model.Project;
+import com.google.refine.operations.OnError;
+import com.google.refine.operations.column.ColumnAdditionOperation;
+
+public class AddColumnCommandTests {
```

Comment:
There's a base class called `CommandTestBase` that the command test classes are expected to extend. This would also remove the need for line 31, where you initialize `request`.